geonode/static/gruntfile.js

```diff
@@ -173,13 +184,68 @@ module.exports = function(grunt) {
       development: {
         src: ['lib/css/*.css'],
         overwrite: true,
-        replacements: [{
-          from: /url\((("?images\/)|('(?!(images|\.)))|(?!('|"))|('\.\.\/images\/))/g,
-          to: 'url(\'../img/'
-        }, {
-          from: /(png|gif|jpg)+(\)|'\)|"\))/g,
-          to: '$1\')'
-        }]
```

Comment:
I changed the pattern in the hope that it will be more readable and understandable.
I also had some issues with the previous pattern didn't replace the text properly for Leaflet.NavBar.css